code/Control/UserDefinedFormController.php

```diff
@@ -327,8 +327,7 @@ public function process($data, $form)
                                 foreach ($validationResult->getMessages() as $message) {
                                     $form->sessionMessage($message['message'], ValidationResult::TYPE_ERROR);
                                 }
-                                Controller::curr()->redirectBack();
-                                return null;
+                                return Controller::curr()->redirectBack();
```

Comment:
```suggestion
                                return $this->redirectBack();
```
Should probably be `$this` since this _is_ a controller.